CampaignConfig/mdc2020_corsika.cfg

```diff
@@ -1,30 +1,31 @@
 [global]
 includes = /exp/mu2e/app/users/sophie/ProductionPush/Production/CampaignConfig/mdc2020_prolog.cfg
-desc = MDC2020aa1
-db_folder = mdc2020aa1
-run_number = 1202
+desc = MDC2020ab
+db_folder = mdc2020ab
+run_number = 1200
```

Comment:
Again we need to apply a consistent run numbering, ideally just to separate Extracted and nominal geometry